res2df/res2csv.py

```diff
@@ -1,38 +1,38 @@
 #!/usr/bin/env python
 """
 End-user command line tool for accessing functionality
-in ecl2df
+in res2df
 """
 import argparse
 import functools
 import importlib
 import sys
 from typing import Optional
 
-from ecl2df import __version__
+from res2df import __version__
 
 # String constants in use for generating ERT forward model documentation:
-DESCRIPTION: str = """Convert Eclipse input and output files into CSV files,
-with the command line utility ``ecl2csv``. Run ``ecl2csv --help`` to see
+DESCRIPTION: str = """Convert reservoir simulator input and output files into CSV files,
+with the command line utility ``res2csv``. Run ``res2csv --help`` to see
 which subcommands are supported.
 
 For supplying options to subcommands, you can use the arguments ``<XARGn>``
 where ``n`` goes from 1 to 10.
 
-For more documentation, see https://equinor.github.io/ecl2df/.
+For more documentation, see https://equinor.github.io/res2df/.
 """
 CATEGORY: str = "utility.eclipse"
 EXAMPLES: str = """
 
-Outputting the EQUIL data from an Eclipse deck. The ECLBASE variable from your
+Outputting the EQUIL data from an input deck. The ECLBASE variable from your
```

Comment:
Instead of saying ECLBASE, we can just say that the it will find the input deck automatically.